sqlalchemy-migrate.requirements

```diff
@@ -0,0 +1,8 @@
+decorator==4.0.10
```

Comment:
What's the reasoning behind putting these in a separate file instead of setup.py?